portfolio/src/main/webapp/task.js

```diff
@@ -153,6 +157,17 @@ function clearTasks() {
   taskList.innerHTML = '';
 }
 
+/**
+ * This method clears inputs from the create new task UI
+ * once a user has added a Task by setting the input values
+ * to empty strings.
+ */
+function clearNewTaskInputs() {
```

Comment:
I think you could also use $('#new-task-name').empty() as well.

Reply:
Done.

Reply:
.empty() would not work here because it empties the inner HTML but what we want it to empty the value since what is written into the input is not stored as: ```<input> User input goes here</input>``` but as ```<input value="User input goes here">```.

Reply:
I see, thanks for letting me know!